### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by ipinak**

### CODE REVIEW

File: articles/unsafe.html [hunk 1]
```diff
@@ -109,7 +109,7 @@ <h3>About the <code>unsafe</code> Standard Package</h3>
 <p>
 Although the return results of calls of any of the three functions
 are consistent in the same program,
-they might be different crossing operation systems of different architectures,
+they might be different crossing operating systems of different architectures,
```
Comment (go101) [inline articles/unsafe.html]: Again, my fault, it should be "crossing different architectures" for preciseness.

File: articles/basic-types-and-value-literals.html [hunk 2]
```diff
@@ -83,7 +83,7 @@ <h3>Built-in Basic Types in Go</h3>
 
 <p>
 The sizes of <code>int</code> and <code>uint</code> values in memory
-are operation system dependent, either 32 or 64 bits.
+are operating system dependent, either 32 or 64 bits.
```
Comment (go101) [inline articles/basic-types-and-value-literals.html]: My fault, it should be "are architecture dependent" for preciseness.

File: articles/struct.html [hunk 3]
```diff
@@ -386,7 +386,7 @@ <h3>About Struct Value Conversions</h3>
 <li>
 	values of type <code>S0</code> can't be converted to
 	the other four types, and vice versa,
-	for the corresponding field names are different.
+	the corresponding field names are different.
```
[the remark this answers is not in the document]
Comment (ipinak) [reply articles/struct.html]: Ok then, it makes more sense now.

File: articles/basic-types-and-value-literals.html (hunk 2 above)
Comment (go101) [reply articles/basic-types-and-value-literals.html]: Maybe, "are operation system and architecture dependent" should be more precise.

File: articles/unsafe.html (hunk 1 above)
Comment (go101) [reply articles/unsafe.html]: Maybe, "are operation system and architecture dependent" should be more precise.

Comment (go101) [reply articles/unsafe.html]: I have changed this place to the following locally.

they might be different crossing operating systems, crossing architectures,
crossing compilers, and crossing compiler versions.

File: articles/basic-types-and-value-literals.html (hunk 2 above)
Comment (go101) [reply articles/basic-types-and-value-literals.html]: I have changed this place to the following locally.

```
The sizes of <code>uintptr</code, <code>int</code> and <code>uint</code> values
n memory are implementation-specific.
Generally, The sizes of <code>int</code> and <code>uint</code> values
are 4 on 32-bit architectures, and 8 on 64-bit architectures.
The size of <code>uintptr</code> value must be large enough to
store the uninterpreted bits of any memory address.
```

File: articles/type-system-overview.html [hunk 4]
```diff
@@ -599,7 +598,7 @@ <h3>Concept: Dynamic Type and Dynamic Value of an Interface Value</h3>
 
 <p>
 For more about interface types and values,
-please read <a href="interface.html">this article</a>.
+please read <a href="interface.html">Interfaces in Go</a>.
```
[the remark this answers is not in the document]
Comment (ipinak) [reply articles/type-system-overview.html]: yes that looks better, thanks :)

File: articles/type-system-overview.html [hunk 5]
```diff
@@ -562,7 +561,7 @@ <h3>Concept: Signature of Function Types</h3>
 
 <p>
 Please read <a href="function.html">functions in Go</a> for more details
-about function type and function values.
+on function type and function values.
```
[the remark this answers is not in the document]
Comment (ipinak) [reply articles/type-system-overview.html]: 👍

File: articles/type-system-overview.html [hunk 6]
```diff
@@ -681,34 +680,31 @@ <h3>Concept: Directions of Channel Types</h3>
...
 
 <a class="anchor" id="types-not-support-comparison"></a>
 <h3>Fact: Types Which Support or Don't Support Comparisons</h3>
 
 <div>
-Currently (Go 1.12), following types don't support comparisons between values of the following types
-(with the <code>==</code> and <code>!=</code> operators):
+Currently (Go 1.12), the following types don't support comparisons between
+values of the following types using the <code>==</code> and <code>!=</code>
+operators:
```
[the remark this answers is not in the document]
Comment (ipinak) [reply articles/type-system-overview.html]: Hehe, it's reading your old code, you are surprised by how it looks.

I agree the new one seems better.

File: articles/type-system-overview.html [hunk 7]
```diff
@@ -681,34 +680,31 @@ <h3>Concept: Directions of Channel Types</h3>
...
-Above listed types are called uncomparable types.
+Above listed types are called incomparable types.
 All other types are called comparable types.
-Compilers forbid comparing two values of uncomparable types.
-</p>
-
-<p>
-Note, some Go tools may call uncomparable types as incomparable types.
+Compilers forbid comparing two values of incomparable types.
 </p>
```
[the remark this answers is not in the document]
Comment (ipinak) [reply articles/type-system-overview.html]: I've fixed this.

File: articles/basic-types-and-value-literals.html (hunk 2 above)
Comment (ipinak) [reply articles/basic-types-and-value-literals.html]: I would change `sizes` to `size`.

File: articles/type-system-overview.html [hunk 8]
```diff
@@ -94,7 +94,7 @@ <h3>Concept: Composite Types</h3>
 </p>
 
 <p>
-<a href="#types-not-support-comparison">Comparable and uncomparable types</a> will be explained below.
+<a href="#types-not-support-comparison">Comparable and incomparable types</a> will be explained below.
 </p>
```
[the remark this answers is not in the document]
Comment (go101) [reply articles/type-system-overview.html]: OK, I will merge this PR soon.
And I will change all  "uncomparable" to "incomparable" in other Go 101 articles.

Thanks very much for these improvements!

**Pull request merged**